src/visualization/types/SimpleTable/PagedTable.tsx

```diff
@@ -285,8 +310,8 @@ const PagedTable: FC<Props> = ({result, properties}) => {
   }, [size]) // eslint-disable-line react-hooks/exhaustive-deps
 
   useEffect(() => {
-    setMaxSize(measurePage(result, 0, height))
-  }, [height, result]) // eslint-disable-line react-hooks/exhaustive-deps
+    setMaxSize(measurePage(result, 0, height, headerHeight, rowHeight))
+  }, [result, height, headerHeight, rowHeight]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
should `setMaxSize` and `measurePage` be included in the dependency array?

Reply:
See above for `measurePage`.

For `setMaxSize`, it's not needed since `setMaxSize` is the setter from a `useState`. As far as I am aware, the setter doesn't change and would make no difference in a dependency array.